build_tools/benchmarks/common/benchmark_driver.py

```diff
@@ -242,3 +246,46 @@ def __get_available_drivers_and_loaders(
       print(f"Available loaders: {available_loaders_str}")
 
     return available_drivers, available_loaders
+
+
+class IreeBenchmarkDriver(BenchmarkDriver):
+
+  def _parse_and_serialize_benchmark_run(self, benchmark_info: BenchmarkInfo,
+                                         results_filename: pathlib.Path,
+                                         benchmark_stdout: str) -> BenchmarkRun:
+    iree_benchmark_json = json.loads(benchmark_stdout)
+    real_times = dict(unit="ns")
+    cpu_times = dict(unit="ns")
+    for metric in ["mean", "median", "stddev"]:
+      real_times[metric], cpu_times[metric] = self._get_aggregate_time(
+          iree_benchmark_json, metric)
+
+    benchmark_run = BenchmarkRun(
+        benchmark_info,
+        context=iree_benchmark_json["context"],
+        real_time=BenchmarkLatency.from_json_object(real_times),
+        cpu_time=BenchmarkLatency.from_json_object(cpu_times),
+    )
+    with open(results_filename, "w") as f:
+      f.write(json.dumps(benchmark_run.to_json_object()))
+    return benchmark_run
+
+  def _get_aggregate_time(self, benchmark_json: Dict[str, Any],
```

Comment:
This helper function seems not to access the `self`. Maybe just define it in the module `benchmark_definition` as a helper function?

Also including `google_benchmarks` in the name might make it clearer that it is used to parse google benchmark results.